### ISSUE TITLE
Streamer_GetAllVisibleItems for CP and RaceCP

### ISSUE THREAD

**Issue by IstuntmanI** [samp-incognito/samp-streamer-plugin#258]
This function is supposed to get ALL visible items. CPs and RaceCPs are visible too, even if only a maximum of one of each. If there's a CP/RaceCP visible, it should be pushed to the `orderedItems` multimap and just fill the array with that one.

I know that we have `GetPlayerVisibleDynamic(Race)CP`, but this can really help for consistency.

I may solve this one too.

**Comment by samp-incognito** [comment]
True, but I'm not sure why you would want to use that native for checkpoints or race checkpoints (you would have to specify an array of size 1) when a more efficient alternative exists. I did make a note on the wiki about that a while ago: https://github.com/samp-incognito/samp-streamer-plugin/wiki/Natives-(Miscellaneous)#Streamer_GetAllVisibleItems

**Comment by IstuntmanI** [comment]
Exactly, just for consistency. I looped through all item types and I made some command to see all nearby/visible (toggleable, if I specify a range of -1.0 it will change to "visible") items and it kinda confused me why there was no (race) checkpoint when it was clear that one was visible. Making an edge case for (race) checkpoints would require more code. This function is also why I requested `Streamer_GetItemPos`.
